wremnants/syst_tools.py

```diff
@@ -2,13 +2,15 @@
 import hist
 import numpy as np
 from utilities import boostHistHelpers as hh, common, logging
+import wremnants
 from wremnants import theory_tools
 from wremnants.datasets.datagroups import Datagroups
 from wremnants.helicity_utils import *
 import re
 import collections.abc
 
 logger = logging.child_logger(__name__)
+narf.clingutils.Declare('#include "theoryTools.h"')
```

Comment:
This should go in wremnants/\_\_init\_\_.py

Reply:
done